src/riak_ql_ddl.erl

```diff
@@ -204,11 +204,11 @@ lk_to_pk(LKVals, Mod, ?DDL{local_key = #key_v1{ast = LKAst},
                            partition_key = PK}) ->
     KeyFields = [F || ?SQL_PARAM{name = [F]} <- LKAst],
     case {length(LKVals), length(KeyFields)} of
-        {_N, _N} ->
```

Comment:
Also, the `_N`s here are _meant_ to match, thus catching the case of `{lengthat (LKVals) == length(KeyFields)`.

What exactly did you try to achieve here, and why?